### PR TITLE
Bug fixes for the styling updates and closest venues

### PR DESCRIPTION
**Pull request by jasvadgama**

### CODE REVIEW

File: src/stylus/venues-search.styl [hunk 1]
```diff
@@ -34,14 +34,17 @@
...
+  top 19px
   right 16px
 
 .venues-search__icon svg
   fill #333
   max-width 24px
   max-height 24px
 
+.venues-search__locateme-wrapper
+    display none // hidden due to chrome not allowing location without https
```
Comment (ecoad) [inline src/stylus/venues-search.styl]: Minor whitespace change needed.

File: src/stylus/venues-marker.styl [hunk 2]
```diff
@@ -1,3 +1,16 @@
 .venues-marker
-  box-shadow 0 8px 6px -6px #000
-  border 5px solid #fff
+  border 1px solid white
+  box-shadow 0 5px 20px 0px rgba(0, 0, 0, 0.5)
+  top -12px // to offset the arrow
+
+  &:after {
```
Comment (ecoad) [inline src/stylus/venues-marker.styl]: No braces needed here

File: src/stylus/venues-search.styl (hunk 1 above)
Comment (jasvadgama) [inline src/stylus/venues-search.styl]: That's been updated

File: src/stylus/venues-marker.styl (hunk 2 above)
Comment (jasvadgama) [inline src/stylus/venues-marker.styl]: They've been removed.

**Pull request merged**